.github/CODEOWNERS

```diff
@@ -1,124 +1,294 @@
 # This file describes the maintainers for the main components. See
 # `dev/doc/MERGING.md`.
 
-# component-path  principal-maintainer secondary-maintainer
+########## GitHub metadata, including this file ##########
 
-# GitHub metadata, including this file
-/.github/         @maximedenes @Zimmi48
+/.github/         @maximedenes
+# Secondary maintainer @Zimmi48
 
-# CI infrastructure
-/dev/ci/*.sh      @ejgallego @SkySkimmer
-/.travis.yml      @ejgallego @SkySkimmer
-/.gitlab-ci.yml   @SkySkimmer @ejgallego
-/.appveyor.yml    @maximedenes @SkySkimmer
+########## CI infrastructure ##########
 
-/default.nix     @Zimmi48 @vbgl
+/dev/ci/*.sh      @ejgallego
+# Secondary maintainer @SkySkimmer
 
-# Documentation
-/README.md        @Zimmi48 @maximedenes
-/INSTALL*         @Zimmi48 @maximedenes
-/CONTRIBUTING.md  @Zimmi48 @maximedenes
-/dev/doc/         @Zimmi48 @maximedenes
-/doc/             @silene @maximedenes
-/man/             @silene @maximedenes
+/.travis.yml      @ejgallego
+# Secondary maintainer @SkySkimmer
 
-/checker/         @barras @maximedenes
+/.gitlab-ci.yml   @SkySkimmer
+# Secondary maintainer @ejgallego
 
-/clib/            @ppedrot @ejgallego
+/.appveyor.yml    @maximedenes
```

Comment:
This is incorrect, appveyor.yml is not a dotfile.

Reply:
Ouch! Fixed, thanks a lot.